config/crd/bases/postgres-operator.crunchydata.com_pgadmins.yaml

```diff
@@ -2165,6 +2170,12 @@ spec:
                               type: string
                             type: array
                         type: object
+                      perDBMetricTargets:
```

Comment:
🤔 This and `customQueries` only make sense at/on Postgres. Is there a way to arrange structs so the PGAdmin API doesn't have these fields? Not a blocker; v1beta1.

Reply:
I didn't want to make any too big changes with the spec, but I wonder if the pgadmin and postgrescluster versions will diverge even more (or if we'll eventually want to slice up instrumentation by target-type, eg., postgres-instrumentation, pgbouncer-instrumentation, pgadmin-instrumentation).

Reply:
Hmm yeah good point... Probably makes sense to at least have a pgadmin-instrumentation and a postgrescluster-instrumentation and then pick and choose which structs make sense in each

Reply:
Should I add "taking apart the instrumentation struct" to this PR?

Reply:
Given the other work due this sprint, I'm going to make a ticket for this topic and not include it in this PR.

Reply:
Made that other ticket.